src/background.js

```diff
@@ -474,10 +478,14 @@ const setup = {
     const runAddonBypassPref = await rollout.getSetting(DOH_SELF_ENABLED_PREF, false);
     const runAddonDoorhangerDecision = await rollout.getSetting(DOH_DOORHANGER_USER_DECISION_PREF, false);
 
+    showConsoleLogs = await browser.experiments.preferences.getBoolPref("doh-rollout.debug", false);
+
+    await log( await browser.storage.local.get() );
```

Comment:
I think this should include some context-setting text. Or alternatively, it seems out of place. Does it belong in this PR?

Reply:
Wrong PR. 🙈